templates/index.html

```diff
@@ -483,7 +475,7 @@ <h1>Oops, payment failed.</h1>
     </div>
   </div>
   </div>
-  <div id="summary">
+  <!-- <div id="summary"> -->
```

Comment:
is this `div` going to be used anywhere? If so, there has to be a comment here describing that, or the `div` has to be removed otherwise.